azure-pipelines/compliance.yml

```diff
@@ -32,3 +32,7 @@ steps:
 - task: CredScan@3
   inputs:
     scanFolder: '$(Build.SourcesDirectory)'
+
+- task: TSAUpload@2
+  displayName: 'TSA upload'
+  GdnPublishTsaConfigFile: '$(Build.SourcesDirectory)/azure-pipelines/TSAOptions.json'

```

Comment:
I _think_ you need this under an `inputs` node to fix the break the pipeline is showing.